ang/sqlTaskConfigurator.js

```diff
@@ -132,15 +132,28 @@
 
                 var title = ts('Task ' + (Number(taskId) ? 'updated' : 'created'));
                 var successMessage = ts('Task successfully ' + (Number(taskId) ? 'updated' : 'created'));
+                var linkToManage = '/sqltasks/manage/' + result.values.id;
+                var linkToRunConfiguration = '/sqltasks/run/configuration/' + result.values.id + '/' + result.values.input_required;
+                successMessage += '<br> <a  href="' + CRM.url('civicrm/a') + '#' + linkToRunConfiguration + '" className="button crm-button">Run Task Now</a>';
+
                 CRM.alert(successMessage, title, 'success');
```

Comment:
Let's add `expiry: 10000, unique: true` as `CRM.alert` options. Civi seems to keep alerts containing links forever; I think it still makes sense to hide them after 10 seconds. `unique: true` makes sense as well as it wouldn't be very useful to see multiple "Task saved" alerts at the same time.